dcicutils/beanstalk_utils.py

```diff
@@ -223,6 +225,35 @@ def swap_cname(src, dest):
     client.restart_app_server(EnvironmentName=dest)
 
 
+def _create_foursight_new(dest_env):
+    """ Helper function that does what create_foursight_auto used to do but slightly differently """
+    fs = {'dest_env': dest_env, 'bs_url': get_beanstalk_real_url(dest_env)}
+
+    # Get information, pass to create_foursight
+    fs['fs_url'] = get_foursight_env(dest_env, fs['bs_url'])
+    fs['es_url'] = ff_utils.get_health_page(ff_env=dest_env)['elasticsearch']
+    fs['foursight'] = create_foursight(**fs)
+
+    # delete initial checks (? not clear why this was happening before)
+    if fs['foursight'].get('initial_checks'):
+        del fs['foursight']['initial_checks']
+
+    return fs
+
+
+def swap_cname(src, dest):
+    """ Does a CNAME swap and foursight configuration (pulled in from Torb)
+        NOTE: this is the mechanism by which CNAME swaps must occur now!
```

Comment:
Now tends to drift. I recommend writing "As of ...date..., this is the preferred mechanism."